Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM ruby:2.4-alpine3.6
+
+LABEL maintainer "olle.jonsson@gmail.com"
```

Comment:
Can you be the maintainer of this Dockerfile? Care to join?

Reply:
Sure! :+1:

Reply:
Done